From 57b2c8dfaefb88b94c153ca99888157360892843 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Thu, 3 Aug 2023 08:53:10 +0800
Subject: [PATCH 2/2] Add packages list for arm64

Add debian_iso_image_arm64.inc from debian_iso_image.inc
and replace qemu-system-x86 with qemu-system-arm.

Story: 2010739
Task: 48533

Depends-On: https://review.opendev.org/c/starlingx/root/+/889682

Change-Id: I59dd20f789de3147297cbffcba1cba287deeb411
Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 debian_iso_image_arm64.inc | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 debian_iso_image_arm64.inc

diff --git a/debian_iso_image_arm64.inc b/debian_iso_image_arm64.inc
new file mode 100644
index 0000000..85cc4e3
--- /dev/null
+++ b/debian_iso_image_arm64.inc
@@ -0,0 +1,16 @@
+#libvirt
+libvirt-clients
+libvirt-daemon
+libvirt-daemon-driver-lxc
+libvirt-daemon-driver-qemu
+libvirt-daemon-driver-storage-gluster
+libvirt-login-shell
+libvirt0
+
+#qemu
+qemu-block-extra
+qemu-guest-agent
+qemu-system-common
+qemu-system-data
+qemu-system-arm
+qemu-utils
-- 
2.30.2

